client/pages/forms/[slug]/show.vue

```diff
@@ -1,166 +1,187 @@
 <template>
-  <div class="bg-white">
+  <div class="bg-[#F5F7F9] w-full">
```

Comment:
If it's a color that's gonna be re-used then we want it it the tailwind config file. Ideally we want to take the closest color from the existing tailwind paletter instead https://v3.tailwindcss.com/docs/customizing-colors